Tools/WebKitTestRunner/InjectedBundle/Bindings/TestRunner.idl

```diff
@@ -371,6 +371,7 @@ interface TestRunner {
     // Storage Access API
     undefined getAllStorageAccessEntries(object callback);
     undefined setRequestStorageAccessThrowsExceptionUntilReload(boolean enabled);
+    Promise<undefined> setStorageAccessPermission(boolean permission, DOMString subFrameURL);
```

Comment:
nit: I think `permission` as a parameter is a little confusing. In this idl other functions just use `boolean value`, and then `bool enabled` in the cpp. I think it would be better to follow the same pattern. Alternatively, `bool granted` instead of `bool permission` may be better.